src/token_request_handler.ts

```diff
@@ -59,7 +59,7 @@ export class BaseTokenRequestHandler implements TokenRequestHandler {
       method: 'POST',
       dataType: 'json',  // adding implicit dataType
       headers: {'Content-Type': 'application/x-www-form-urlencoded'},
```

Comment:
Just drop the `header` from the dictionary. The implied request content type is `application/json`.

Reply:
The [OAuth2.0 Token Revocation spec](https://tools.ietf.org/html/rfc7009#section-2.1) calls for `application/x-www-form-urlencoded`.  

IdentityServer4 [enforces this](https://github.com/IdentityServer/IdentityServer4/blob/a68af9dcc0fb5b57c0ae6921670b5b56b16cb376/src/Endpoints/TokenRevocationEndpoint.cs#L69) as well.

Reply:
If that’s the case. Drop the `dataType: json`. Let me take a look at the spec as well.

Reply:
@tikurahul the `dataType` property has been removed